ts/components/Pinpad/index.tsx

```diff
@@ -125,13 +152,26 @@ class Pinpad extends React.PureComponent<Props, State> {
   public clear = () => this.setState({ value: "" });
 
   public render() {
+    // animation interpolate from top to down and the other way around
+    const marginTopDown = this.animatedValue.interpolate({
+      inputRange: [0, 0.2, 0.4, 0.6, 0.8, 0.9, 1],
+      outputRange: [0, -10, 10, -10, 10, -10, 0]
+    });
     return (
       <React.Fragment>
-        <View style={styles.placeholderContainer}>
-          {this.placeholderPositions.map(this.renderPlaceholder)}
-        </View>
-        <View spacer={true} extralarge={true} />
-        <KeyPad digits={this.pinPadDigits} buttonType={this.props.buttonType} />
+        <Animated.View
```

Comment:
Would be nice to have a `<ShakeAnimation>` component that is reusable and decoupled from the `PinPad`.